src/pages/CreateFeedback.tsx

```diff
@@ -51,54 +72,73 @@ export const CreateFeedback = (props: IProps) => {
       <img className="add" src={newFeedback} alt="plus sign" />
       <form onSubmit={handleSubmit(onSubmit)}>
         <h1>Create New Feedback</h1>
-        <div>
+        <div className="container">
           <label htmlFor="titleRequired">
             <span>Feedback Title</span>
             <span>Add a short, descriptive headline</span>
           </label>
-          {errors.titleRequired && (
-            <span className="error">Can't be empty.</span>
-          )}
+          {errors.title && <span className="error">Can't be empty.</span>}
           <input
             type="text"
-            {...register('titleRequired', { required: true })}
+            className="text"
+            {...register('title', { required: true })}
           />
         </div>
 
         <label htmlFor="category">
           <span>Category</span>
           <span>Choose a category for your feedback</span>
         </label>
-        <select id="category" {...register('category')}>
-          <option value="feature">Feature</option>
-          <option value="bug">Bug</option>
-          <option value="enhancement">Enhancement</option>
-          <option value="ui">UI</option>
-          <option value="ux">UX</option>
-        </select>
+        <div className="dropdown">
+          <Controller
+            name="category"
+            control={control}
+            rules={{
+              required: true,
+            }}
+            render={({ field: { onChange, value, name, ref } }) => {
+              const currentSelection = categoryOptions.find(
+                (c: LabelValue) => c.value === value,
+              );
 
-        <div>
-          <label htmlFor="feedbackRequired">
+              return (
+                <Select
+                  name={name}
+                  options={categoryOptions}
+                  onChange={(selectedOption: LabelValue | null): void => {
+                    onChange(selectedOption?.value);
+                  }}
+                  ref={ref}
+                  value={currentSelection}
+                  isSearchable={isSearchable}
```

Comment:
Typically we only `setState` if we _need_ to.

For these props, you're never using the `setIsSearchable` (for example) - IE, never toggling the `isSearchable` prop.

So, I would say you don't need a `useState`! These can "hardcoded" because they're never changing.

Reply:
Removed!